AutorestSwiftTest/AutoRestReportTest.swift

```diff
@@ -0,0 +1,85 @@
+// --------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// The MIT License (MIT)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the ""Software""), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import Foundation
+import XCTest
+import AzureCore
+import AutoRestReport
+
+class AutoRestReportTest: XCTestCase {
```

Comment:
So it looks like we are testing out ability to retrieve the report... but are we actually doing anything with the results?

Reply:
Yes, getting the report is the first step. Need to follow the rest of the steps from Laurent.

Reply:
Do those steps take place in here or are they an outside thing?

Reply:
Outside of this test case.

Reply:
Actually, I was wrong. i look at the test report in Python side, it `print_report` after it gets the report https://github.com/Azure/autorest.python/blob/autorestv3/test/vanilla/AcceptanceTests/test_zzz.py.